src/views/diff.rs

```diff
@@ -300,10 +306,83 @@ pub fn SourceView(props: &SourceViewProps) -> Html {
                         />
                     </nav>
                     <div id="diff-view" class="flex-1">
-                        <DiffView {diff} path={props.path.clone()} />
+                    { for (*files).iter().map(|path| html!{<LazyDiffView diff={diff.clone()} path={path.clone()} /> }) }
                     </div>
                 </main>
             </Content>
         </>
     }
 }
+
+fn determine_display_files(props: &FileTreeProps) -> Vec<Utf8PathBuf> {
```

Comment:
not sure if best solution - would be faster to krey through filepaths for `^path($|\/)`